scripts/Install-AllSoftware.ps1

```diff
@@ -0,0 +1,23 @@
+# 1. Make sure the Microsoft App Installer is installed:
+#    https://www.microsoft.com/p/app-installer/9nblggh4nns1
+# 2. Run this script as administrator. If running scripts is blocked, you can temporarily unblock them by running 
+#    Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
+
+Write-Output "Installing apps"
+$apps = @(
+	@{packageID = "icsharpcode.ILSpy" },
+	@{packageID = "Microsoft.DotNet.SDK.6" },
+   	@{packageID = "Microsoft.VisualStudioCode" }
+);
+Foreach ($app in $apps) {
+    $listApp = winget list --exact -q $app.packageIDs
+    if (![String]::Join("", $listApp).Contains($app.packageID)) {
+        Write-output "Installing: " $app.packageID
+        winget install -h $app.packageID -s winget 
+    }
+    else {
+        Write-output "Skipping: " $app.packageID " (already installed)"
```

Comment:
Instead of outputting 3 separate strings into the pipeline, we can use the [subexpression operator](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.2#subexpression-operator--) to get that package ID into the string.

```suggestion
        Write-Output "Skipping: $($app.packageID) (already installed)"
```

Reply:
Makes sense! Made this modification to the other output line too.